src/use-cases/queries/RecupererMesMembres.ts

```diff
@@ -0,0 +1,80 @@
+import { QueryHandler } from '../QueryHandler'
+import { UnaryOperator } from '@/shared/lang'
+
+export class RecupererMesMembres implements QueryHandler<Query, MesMembresReadModel> {
+  readonly #mesMembresLoader: MesMembresLoader
+
+  constructor(mesMembresLoader: MesMembresLoader) {
+    this.#mesMembresLoader = mesMembresLoader
+  }
+
+  async get(query: Query): Promise<MesMembresReadModel> {
+    return this.#mesMembresLoader.findMesMembres(query.codeDepartement, (mesMembres) => {
+      const membres = preFiltrageStatut(mesMembres.membres, query.statut)
+      return {
+        ...mesMembres,
+        filtres: filtresDistinct(membres),
+        membres,
+        statut: statutDistinct(mesMembres.membres),
+      }
+    })
+  }
+}
+
+export abstract class MesMembresLoader {
+  async findMesMembres(
+    codeDepartement: string,
+    filtrations: UnaryOperator<MesMembresReadModel>
```

Comment:
On peut rester générique ici, le terme "filtration" veut dire autre chose.
```suggestion
    operator: UnaryOperator<MesMembresReadModel>
```